tests/smoke/events.js

```diff
@@ -105,46 +105,48 @@
 				editor = bot.editor,
 				resumeAfter = bender.tools.resumeAfter,
 				observer = observeSpellCheckEvents(editor),
-				starterHtml = '<ol><li id="a">something<ul><li id="b">somethingnested^</li></ul></li></ol>';
+				starterHtml = '<ol><li id="a">something<ul><li id="b">somethingnested blah^</li></ul></li></ol>';
 
-				bot.setHtmlWithSelection(starterHtml);
+			// the second list item must have two words, because the word at caret is ignored
```

Comment:
this function is mostly indent fixes + adding an extra words in the starting HTML.  If no caret is provided, setHtmlWithSelection puts the caret in different places depending on browser, and can cause the test to fail in certain browsers.